src/styles/cards.scss

```
@@ -0,0 +1,124 @@
+.main__container {
+  display: flex;
+  flex-wrap: wrap;
+  width: (4 * $card-width) + (3 * $card-margin) + (2 * $inside-padding);
+  margin: 0 auto;
+  padding-left: 2px;
+}
+
+p {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: $white;
+}
+
+.card {
+  background-color: var(--white-color);
+  max-width: $card-width;
+  border: 1px solid #f3f3f3;
```

don't forget to create variables for all colors
```suggestion
  border: 1px solid $c-color;
```